generators/server/templates/quarkus/src/test/java/package/web/rest/UserResourceTest.java.ejs

```diff
@@ -162,7 +166,15 @@ public class UserResourceTest {
             .when()
             .post("/api/users")
             .then()
-            .statusCode(BAD_REQUEST.getStatusCode());
+            .statusCode(BAD_REQUEST.getStatusCode())
+            .header("X-" + applicationName + "-error", is("error.idexists"))
+            .header("X-" + applicationName + "-params", is("userManagement"))
+            .contentType("application/problem+json")
+            .body("title", is("A new user cannot already have an ID"))
+            .body("entityName", is("userManagement"))
```

Comment:
I'm also not super sure if all those fields (entityName, errorKey, message, params) are needed/used by frontend - I used springboot version as reference and added everything I could find there.

Reply:
They do